app/helpers/react_components/dropdowns/dropdown.rb

```diff
@@ -0,0 +1,23 @@
+module ReactComponents
+  module Dropdowns
+    class Dropdown < ReactComponent
+      def initialize(menu_button:, menu_items:)
+        super()
+
+        @menu_button = menu_button
+        @menu_button[:id] = "rnd-#{SecureRandom.hex(5)}"
```

Comment:
I want to be able to have multiple components on the same page so don't want to use IDs. I understand that the IDs are required for a11y, so I'm just setting a random ID instead. I think this could be moved within React instead of being set each time here.